multipaz-mlkit/src/androidMain/kotlin/org/multipaz/facematch/FaceMatchLiteRtModel.android.kt

```diff
@@ -0,0 +1,84 @@
+package org.multipaz.facematch
+
+import android.content.Context
+import org.jetbrains.compose.resources.InternalResourceApi
+import org.multipaz.util.Logger
+import java.io.File
+import java.io.FileOutputStream
+import java.nio.ByteBuffer
+
+private const val TAG = "LiteRtModel"
+
+object AppContextHolder {
```

Comment:
Where is this used? If you need the application context, just use [`applicationContext`](https://github.com/openwallet-foundation-labs/identity-credential/blob/main/multipaz/src/androidMain/kotlin/org/multipaz/context/utils.kt) - we require Multipaz applications to initialize this with `initializeApplication()`...

Also, it should probably be `internal`.